### PR TITLE
2.10.0 - New menu system

### LINKED ISSUE #385: InputPopup steals input devices and never gives them back
If I open a custom input popup from the save menu I see the popup text and the pause menu text at the same time. I interact with both, which is an issue unto itself. Once both the popup and the pause menu are closed I can no longer look around with mouse or gamepad. I'm stuck in the mouse mode as if the input still belongs to `ModHelper.Menus` and not the game. The only way out is to quit the game. Full bork.

The calling code is doing this:
`var popup = ModHelper.Menus.PopupManager.CreateInputPopup`

### LINKED ISSUE #500: Default settings button has wrong text
image

Shows the text from the last base game tab you were on. Here I went from GRAPHICS to MODS to the settings for Custom Ship Log Modes, but it still says I will be changing to default settings for graphics.

### LINKED ISSUE #514: "Leave menu" in mod config menu goes back to the main menu, instead of the MODS tab as expected

### LINKED ISSUE #515: Some buttons in mod menus don't make any noise

### PR DESCRIPTION
**Pull request by misternebula**
# OWML 2.10.0

## New Menus
- Introduced a whole new, easier to use, menu system.
   - Old menu system is still availabe for backwards compatibility.
   - New menu system is accessed through `ModHelper.MenuHelper`. Old menus are still located at `ModHelper.Menus`.

## Smaller Changes
- Added DefaultConfig to ModHelper.
- Changed logs in ModAssets to be MessageType.Debug (removes "Loading asset bundle from ... " spam unless you have OWML in debug mode)
- Added full OW version and OWML version to title screen

Fixes #515
Fixes #514 
Fixes #500 
Fixes #385 (?)

- [x] controller inputs on popups
  - inputs only don't work on input popup. but you need keyboard for that anyway sooo
- [x] can't go up from text input, only down

### CODE REVIEW

**Comment by MegaPiggy** [comment]
👀

**Comment by Bwc9876** [comment]
Would writing documentation for this on the docs site be something you're willing to do?

**Comment by misternebula** [comment]
Yup!

**Review by MegaPiggy — APPROVED** [review]

File: src/OWML.ModHelper.Menus/ModMenus.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ private void InitMainMenu(TitleScreenManager titleScreenManager)
 				MainMenu.Initialize(titleScreenManager);
 				var inputMenu = titleScreenManager.GetComponent<ProfileMenuManager>().GetValue<PopupInputMenu>("_createProfileConfirmPopup");
 				PopupManager.Initialize(inputMenu, MainMenu.OptionsMenu);
-				ModsMenu.Initialize(this, MainMenu);
+				//ModsMenu.Initialize(this, MainMenu);
```
Comment (Raicuparta) [inline src/OWML.ModHelper.Menus/ModMenus.cs:62]: Leftover commented-out code

**Pull request merged**